action/Action/DrawManager.h

```diff
@@ -8,3 +8,21 @@ void DrawEnd();		// 
 
 void DrawInformation();									// ��񕶏͂�`��
 void DrawGameClear();									// �Q�[���N���A�̕�����`��
+
+class Fade
+{
+public:
+	int m_alpha;
+	bool m_nowFading;
```

Comment:
ブサイクです。
```suggestion
	bool m_isFading;
```
はいかがでしょう。

Reply:
提案して頂いたものに変更しました。
2861269a